posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<p>This section feels a bit light compared to the rest of the notebook. It kind of feels like you've introduced Mojo and then told us not to use it. Is it necessary to mention at all? Is a Mojo implementation of interp much faster?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>